placeholder_accessors/index.md

```diff
@@ -192,6 +193,51 @@ The handler gains a new method,
 that registers the requirements of the placeholder accessor on the given
 command group.
 
+### Placeholder `accessor` without a buffer
+
+Not providing either the handler or the memory object
+makes the accessor default constructible,
+creating a "null" accessor.
+A null accessor can be bound to a buffer
+by constructing a new placeholder accessor with the buffer
+and assigning it to the null accessor.
+Example:
+
+```cpp
+accessor<T, dim, mode, target, access::placeholer::true_t> acc0;
+
+acc0 = accessor<T, dim, mode, target, access::placeholer::true_t>(buf);
+```
+
+To simplify placeholder accessor construction,
+a new free function `cl::sycl::codeplay::make_placeholder_accessor` is provided,
+which takes a buffer as an argument:
+
+```cpp
+template <access::mode mode, access::target target, typename T, int dim,
+          typename AllocatorT>
+accessor<T, dim, mode, target, access::placeholder::true_t>
+make_placeholder_accessor(buffer<T, dim, AllocatorT>& bufferRef);
+```
+
+A null accessor does not need to be registered with the command group,
+i.e. there is no need to call `require()` on it.
+It is allowed to pass a null accessor to a kernel,
+as long as it's not dereferenced inside the kernel,
+otherwise an out-of-bounds access will occur,
+likely leading to a segmentation fault or similar.
```

Comment:
This is quite ambiguous. Is better simply to say a device-specific asynchronous error